app/src/main/res/values/arrays.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
if there is nothing in this file then why did you add arrays.xml in your commit?